README.md

```diff
@@ -63,6 +63,7 @@ feel free to contact us in English or German.
 This project is funded by [Stiftung für Innovation in der Hochschullehre](https://stiftung-hochschullehre.de)
 (["Freiraum 2022"](https://stiftung-hochschullehre.de/foerderung/freiraum2022/)).
 
+The used [Assets](doc/wiki/assets-im-dungeon.md) are blended from free resources.
```

Comment:
siehe https://github.com/Programmiermethoden/Dungeon/pull/1285#discussion_r1410300559: das ist zu indirekt und zu unspezifisch. hier gehört noch der ordner mit her und die einzelnen links und die angabe cc0 in klammern ...